integration/__specs__/modules/components/cartPage/cartModal/cartQuantityInput/cartQuantityInput.ts

```diff
@@ -0,0 +1,11 @@
+import { Component } from '@Core/component';
+
+export class CartQuantityInput extends Component {
+    protected selectors = {
+        quantityInput: './/input[@name="quantity"]',
+    };
+
+    public async getNameInput() {
+        await this.element.waitForXpath(this.selectors.quantityInput);
+    }
+}
```

Comment:
We don't need this component, since we create them using 'Component' in cartModal.ts